plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -140,4 +154,74 @@ public function get_order_item_type( $order, $order_item_id ) {
 	}
 
 	//phpcs:enable Squiz.Commenting.FunctionComment.Missing
+
+
+	public function get_schema() {
+		$orders_table_name = $this->get_orders_table_name();
+		$addresses_table_name = $this->get_addresses_table_name();
+		$operational_data_table_name = $this->get_operational_data_table_name();
+		$table = "
+CREATE TABLE $orders_table_name (
+	id bigint(20) unsigned auto_increment,
+	post_id bigint(20) unsigned null,
+	status varchar(20) null,
+	currency varchar(10) null,
+	tax_amount decimal(26,8) null,
+	total_amount decimal(26,8) null,
+	customer_id bigint(20) unsigned null,
+	billing_email varchar(320) null,
```

Comment:
When somebody use INNODB utf8mb4 this value exceeds the limit. In this case the max accepted VARCHAR length is 191. Otherwise it fails with "Specified key was too long; max key length is 1000 bytes" error.